composer.json

```diff
@@ -11,32 +11,33 @@
         {"name":"Lukas Kahwe Smith", "email":"smith@pooteeweet.org"}
     ],
     "minimum-stability": "dev",
+    "prefer-stable": true,
     "require": {
-        "php": ">=5.3.9",
+        "php": "^5.3.9|~7.0",
         "doctrine/common": "~2.4",
         "doctrine/annotations": "~1.2",
-        "doctrine/data-fixtures": "~1.0,>=1.0.0",
+        "doctrine/data-fixtures": "~1.0",
         "phpcr/phpcr": "~2.1.1",
         "phpcr/phpcr-implementation": "~2.1.0",
         "phpcr/phpcr-utils": "^1.2.8",
-        "doctrine/instantiator": "~1.0.1",
+        "doctrine/instantiator": "^1.0.1",
         "symfony/console": "~2.3|~3.0"
     },
     "require-dev": {
         "symfony/yaml": "~2.3|~3.0",
-        "symfony/phpunit-bridge": "2.7.*@dev",
+        "symfony/phpunit-bridge": "~2.7|~3.0",
         "liip/rmt": "~1.1"
     },
     "suggest": {
         "symfony/yaml": "~2.3|~3.0",
-        "jackalope/jackalope-doctrine-dbal": "~1.1",
+        "jackalope/jackalope-doctrine-dbal": "^1.1.5",
         "jackalope/jackalope-jackrabbit": "~1.0"
     },
     "conflict": {
         "jackalope/jackalope-doctrine-dbal": "<1.1.5"
     },
     "autoload": {
-        "psr-0": { "Doctrine\\ODM\\PHPCR": "lib/" }
+        "psr-4": { "Doctrine\\ODM\\PHPCR\\": "lib/Doctrine/ODM/PHPCR" }
```

Comment:
oh, actually we should probably have moved the code up to lib/ as well and get rid of the extra folders.